### PR TITLE
add ES translation of AngularJS Waypoints

### PR DESCRIPTION
**Pull request by justinian336**
Translation to Spanish of the Angular.js Waypoints.

### CODE REVIEW

File: seed/challenges/angularjs.json [hunk 1]
```diff
@@ -137,8 +152,12 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Crea servicios con Angular.js",
+      "descriptionEs": [
+        "Servicios son funciones que puedes usar y reciclar en tu app en Angular.js para mejorar tu eficiencia.",
```
Comment (vtamara) [inline seed/challenges/angularjs.json]: Agregar "Los" antes de "servicios"

**Comment by vtamara** [comment]
Good job with this translation.

File: seed/challenges/angularjs.json (hunk 1 above)
Comment (justinian336) [inline seed/challenges/angularjs.json]: Tienes razón, suena mejor así.

**Comment by justinian336** [comment]
Will leave it like this by now and then patch it once the rest of the translation is done.

**Comment by QuincyLarson** [comment]
Great work here, @justinian336!

**Pull request merged**